ush/ufsda/misc_utils.py

```diff
@@ -56,20 +56,23 @@ def create_batch_job(job_config, working_dir, executable, yaml_path):
     with open(batch_script, 'w') as f:
         f.write('#!/bin/bash\n')
         if scheduler[job_config['machine']] == 'slurm':
+            if job_config['machine'] == 'hera':
+                taskspernode = job_config.get("ntasks-per-node", 18)
+            elif job_config['machine'] == 'orion':
+                taskspernode = job_config.get("ntasks-per-node", 24)
+            else:
+                taskspernode = job_config.get("ntasks-per-node", 12)
             sbatch = f"""#SBATCH -J GDASApp
 #SBATCH -o GDASApp.o%J
 #SBATCH -A {job_config['account']}
 #SBATCH -q {job_config['queue']}
 #SBATCH -p {job_config['partition']}
 #SBATCH --ntasks={job_config['ntasks']}
+#SBATCH --ntasks-per-node={taskspernode}
```

Comment:
We should add logic to ensure the combination of `taskspernode` and `cpu-per-task` does not oversubscribe the node.   This requires us to know the maximum number of processes per node.

Reply:
Yes, I run JEDI apps with OMP_NUM_THREADS=1.   This is a serious weakness of JEDI in its current form.   I hope (expect) this limitation to be removed at some point.   Your reply raises a good point.  When users see `cpus-per-tasks` in the configuration yamls, they may think we can run JEDI with multiple threads.   This is not advisable at present.  Hard wiring `cpus-per-task` to 1 is the safest thing to do now.   We'll just need to remember to undo this change once threading is safely added to JEDI.

Reply:
ok let me update this PR with removing it from the YAMLs and hard wiring it for now

Reply:
only took a minute, and I think it's the right call, don't want to make things confusing for the users